src/main/java/com/vesoft/nebula/jdbc/NebulaAbstractPreparedStatement.java

```diff
@@ -58,7 +61,9 @@ protected void insertParameter(int parameterIndex, Object obj) throws SQLExcepti
 
     @Override
     public ResultSetMetaData getMetaData() throws SQLException {
-        return this.getMetaData();
+
+        ResultSet currentResultSet = this.currentResultSet;
+        return currentResultSet.getMetaData();
```

Comment:
`currentResultSet` will be null if the PrepareStatement has not been executed, so null check is need before `currentResultSet.getMetaData()`.